### PR TITLE
feat: add new ECR endpoints to EKS ECR credential provider config

### PR DESCRIPTION
**Pull request by BDworak**
**Issue #, if available:**

**Description of changes:**
Add new upcoming ECR endpoints to EKS ECR Credential provider configuration json.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

**Testing Done**
No testing, this is just a configuration update.

### CODE REVIEW

**Comment by BDworak** [comment]
We should hold off on merging this PR until the cloud-provider-aws PR gets merged to maintain correct order of operations: https://github.com/kubernetes/cloud-provider-aws/pull/1069.

**Review by ndbaker1 — APPROVED** [review]

**Pull request merged**
